src/czbenchmarks/datasets/dataset.py

```diff
@@ -0,0 +1,90 @@
+from abc import ABC, abstractmethod
+from pathlib import Path
+from typing import IO, Any
+import os
+
+from .types import Organism
+
+
+class Dataset(ABC):
+    """
+    A base class for task-specific datasets. Each concrete dataset class will extract the data required for a specific type of task from the provided input file. These task specific data items can then be retrieved as object instance variables or written to files for later use.
+    
+    path: Path
```

Comment:
is this left over from some earlier revision maybe? It's identical to line 20 but seems out of place